supervisor/const.py

```diff
@@ -38,6 +38,7 @@
 DOCKER_NETWORK = "hassio"
 DOCKER_NETWORK_MASK = ip_network("172.30.32.0/23")
 DOCKER_NETWORK_RANGE = ip_network("172.30.33.0/24")
+DOCKER_NETWORK_LINK_LOCAL = ip_network("fd00:172:30:32::/64")
```

Comment:
Note that this is a unique local address (ULA), which is not the same as a link local address (which start with `fe80`). Link local addresses can not be routed, unique local addresses can be routed in private networks (used to be called site).

I am not sure if we could get away with link local addressing only (which gets assigned by default when IPv6 is enabled), but definitely doesn't hurt to add a unique local address.

Reply:
It will work. It looks like we need add for now `  "experimental": true, "ip6tables": true` to dockerd for support nat6

Reply:
https://github.com/robbertkl/docker-ipv6nat/issues/65#issuecomment-883485635

Reply:
Just to be clear, I suggest to change the name of that constant, since its a ULA, e.g.

```suggestion
DOCKER_NETWORK_ULA = ip_network("fd00:172:30:32::/64")
```

> It will work.

With just link local addressing (aka, not setting any address manually) or with unique local addressing?